eslzArm/eslz-portal.json

```
@@ -2032,6 +2032,298 @@
                                     }                                   
                                 ]
                             }
+                        },
+                        {
+                            "name": "vWANHubRoutingPreference",
+                            "type": "Microsoft.Common.DropDown",
+                            "label": "Hub Routing Preference",
+                            "defaultValue": "ExpressRoute",
+                            "multiselect": false,
+                            "selectAll": false,
+                            "filter": false,
+                            "multiLine": true,
+                            "visible":"[and(not(equals(steps('connectivity').enableHub, 'No')), not(equals(steps('connectivity').enableHub, 'nva')), not(equals(steps('connectivity').enableHub, 'vhub')))]",
+                            "toolTip":"Preference used in selecting best path when the virtual hub learns multiple paths to the same destination route-prefix.<a href=\"https://learn.microsoft.com/en-us/azure/virtual-wan/about-virtual-hub-routing-preference\">Virtual hub routing preference</a>.",
```

```suggestion
                            "toolTip":"Preference used in selecting best path when the virtual hub learns multiple paths to the same destination route-prefix.<a href=\"https://learn.microsoft.com/azure/virtual-wan/about-virtual-hub-routing-preference\">Virtual hub routing preference</a>.",
```

Removing the fixed language to make this more D&I